tests/EachPromiseTest.php

```diff
@@ -323,6 +323,10 @@ private function createSelfResolvingPromise($value)
 
     public function testMutexPreventsGeneratorRecursion()
     {
+        if (defined('HHVM_VERSION')) {
+            $this->markTestIncomplete('Broken on HHVM.');
```

Comment:
Why "add support back for hhvm" when most of the tests fail? Then it obviously isn't supported.

Reply:
Most of the tests actually pass. Only 19 of 255 needed to be skipped.